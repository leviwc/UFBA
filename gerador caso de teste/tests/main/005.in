27 88
9 9
10 21
24 6
16 26
5 22
13 25
21 2
9 18
21 15
1 3
22 1
6 5
26 8
24 23
5 11
6 6
22 17
3 23
1 14
18 13
1 11
22 20
4 23
26 26
18 22
19 22
17 27
2 4
25 18
16 24
14 20
5 1
2 2
26 6
17 17
5 7
10 18
9 16
6 20
25 21
11 20
5 25
19 27
20 19
20 10
6 13
17 14
23 20
25 14
9 14
12 9
6 2
18 4
4 15
11 14
19 21
1 10
11 10
27 2
12 21
21 6
1 23
8 11
4 10
15 7
25 22
24 19
25 26
8 20
22 7
16 25
1 21
19 6
1 17
7 17
11 15
4 26
14 26
13 9
3 18
17 25
19 16
23 15
18 7
2 17
22 3
19 8
21 14
