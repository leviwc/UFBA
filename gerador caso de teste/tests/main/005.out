Resultado com busca binaria: 10
3105.673000 s
Resultado com backtracking apenas: 10
2.614000 s
Resultado testando todas as possibilidades: 10
6667.569000 s
Resultado com busca binaria otimizada: 10
2466.423000 s
